pkg/util/indexparamcheck/bitmap_checker_test.go

```diff
@@ -0,0 +1,21 @@
+package indexparamcheck
+
+import (
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+
+	"github.com/milvus-io/milvus-proto/go-api/v2/schemapb"
+)
+
+func Test_BitmapIndexChecker(t *testing.T) {
```

Comment:
I think it's better to rename this file to "bitmap_index_checker_test.go".

Reply:
actually others checkers not add index name, follow the same rule

Reply:
OK.